style.css

```diff
@@ -85,12 +95,127 @@ body {
   width: 200px;
   display: flex;
   justify-content: space-between;
-  padding-bottom: 325px;
-  padding-left: 6%;
+  padding-left: 4%;
+  padding-bottom: 50px;
 }
 
 .social {
   font-size: 300%;
   color: #505f79;
   text-decoration: none;
 }
+
+.work-container {
+  padding-top: 200px;
+  display: grid;
+  gap: 88px;
+  background: #e5e5e5;
+}
+
+.work-item-flex {
+  width: 90%;
+  padding: 4% 4% 4% 4%;
+  margin: 5% 5% 5% 5%;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+}
+
+.work-item-title {
+  margin-top: 4%;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 600%;
+  line-height: 100%;
+  color: #172b4d;
+}
```

Comment:
As the two previous reviewers said, kindly make sure you follow the Figma design sizing, and fonts properly since that is the main objective of this project. For instance the font-weght here is 900, but it is supposed to be 700 as shown below in the figma design below. Your font-size is 600% (96px) but it should be 32px (200%) as shown below. It is clear there are some huge differences between your design and the figma designs for most parts of your project. I see the problem stems from the fact that you try to use a lot of percentages(%) instead of pixels (px), there is nothing wrong with that, but make sure your conversion from px to % is always accurate, else it will cause you a lot of problems. Please try as much as possible to correct this discrepancies and conform to the provided figma designs as it will really go along way to help you.

![Screenshot from 2022-09-23 10-48-16](https://user-images.githubusercontent.com/65233860/191944981-38a82afa-5801-4235-827c-a36834c96b04.png)

Reply:
I've just changes all font-size, font-weght, margins etc to follow Figma. But now my design of "work" pages look weird, all text is too small with 375px width. In the same time, the 1st page and "about" page looks fine. What is wrong with "work" pages, could you please advise?

Reply:
To solve this issue with text size, I've found command "text-size-adjust: 300%;", now it looks better.